code/modules/clothing/suits/ego_gear/waw.dm

```diff
@@ -394,7 +396,7 @@ Any attempt to code risk class armor will result in a 10 day Github ban.*/
 	name = "rimeshank"
 	desc = "Well, I can't just shiver in the cold forever, can I?"
 	icon_state = "rimeshank"
-	armor = list(RED_DAMAGE = 70, WHITE_DAMAGE = 40, BLACK_DAMAGE = 0, PALE_DAMAGE = 30) //140
+	armor = list(RED_DAMAGE = 70, WHITE_DAMAGE = 40, BLACK_DAMAGE = 0, PALE_DAMAGE = 30, FIRE = 20) //140
```

Comment:
Why does a Waw have a fire resistance of a zayin?

Reply:
If it was zayin it would have a defense of 10, but it has low fire resistance because its armor made of ice. Just a bit of flavor.